src/app/ui-kit/components/question-item/question-item.component.scss

```diff
@@ -0,0 +1,54 @@
+$color1: #fff;
+$color2: #e23d69;
+
+.radio {
+  align-items: center;
+  display: flex;
+  margin: 10px;
+
+  input[type='radio'] {
+    opacity: 0;
+    position: absolute;
+
+    + .radio-label {
+      display: flex;
+
+      &:before {
+        background: theme('colors.bright');
+        border: 2px solid theme('colors.shade');
+        border-radius: 100%;
+        content: '';
+        cursor: pointer;
+        display: inline-block;
+        height: 1.4em;
+        margin-right: 10px;
+        text-align: center;
+        transition: all 250ms ease;
+        width: 1.4em;
+      }
+    }
+
+    &:checked {
+      + .radio-label {
+        color: theme('colors.bright');
+
+        &:before {
+          background-color: theme('colors.error');
+          border-color: theme('colors.error');
+          box-shadow: inset 0 0 0 4px theme('colors.bright');
+        }
+      }
+    }
+
+    &:focus {
+      + .radio-label {
+        color: theme('colors.bright');
+
+        &:before {
+          border-color: theme('colors.error');
+          outline: none;
+        }
+      }
+    }
+  }
+}
```

Comment:
in a separate radio component, let's overview what styles can be applied by tailwind.